### PR TITLE
Add folder for CNDT 2022

### PR DESCRIPTION
**Pull request by atoato88**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
This PR adds the assets for holding k8s training oh CloudNative Days Tokyo 2022.

**Which issue(s) this PR fixes**:
NONE

**Special notes for your reviewer**:
NONE

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:
NONE

### CODE REVIEW

**Comment by atoato88** [comment]
/assign @oomichi 
/assign @shu-mutou 
Could you review this?

File: japan/cndt-2022/OWNERS [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  - s-kawamura-w664
+  - t-inu
+  - ziyi-xie
+approvers:
+  - oomichi
+  - atoato88
+  - shu-mutou
+  - s-kawamura-w664
+  - t-inu
+  - ziyi-xie
```
Comment (oomichi) [inline japan/cndt-2022/OWNERS:16]: Why do we need to add OWNERS file for each event?
I guess `japan/OWNERS` file could be enough for all events.
Or do we want to clarify who are instructors from each OWNERS file for each event?

Comment (atoato88) [reply japan/cndt-2022/OWNERS:16]: +1
I agree with your comment.
I'll remove OWNERS in this folder from PR.

Comment (atoato88) [reply japan/cndt-2022/OWNERS]: done.

**Pull request merged**